virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -8,6 +8,8 @@ toc::[]
 
 {VirtProductName} provides advanced networking functionality by using custom resources and plugins. Virtual machines (VMs) are integrated with {product-title} networking and its ecosystem.
 
+image::318_OpenShift_Virtualization_Networking_0423.png[{VirtProductName} Networking architecture]
+
```

Comment:
Suggested text (to explain the basic concepts, but to make sure people understand this is not the only solution we have to offer):

Illustration of a typical network setup of OpenShift Virtualization. Other options are available.

1. Pods and VMs are running on the same network infrastructure, allowing users to easily connect their containerized and virtualized workloads
2. VMs can be connected to the default pod network and to any number of secondary networks
3. The default pod network provides connectivity between all its members, service abstraction, it supports IP management, micro segmentation, and more
4. Multus is a component allowing users to connect to secondary networks
5. The default pod network is overlay-based, tunneled through the underlying machine network
6. The machine network can be defined over a selected set of NICs
7. Secondary VM networks are typically bridged directly to a physical network, with or without VLAN encapsulation
8. Secondary VM networks can be defined on dedicated set of NICs (in the picture) or they can use the machine network